lib/auctionrunner.js

```diff
@@ -0,0 +1,257 @@
+/*
+ * Auction Runner
+ */
+
+'use strict';
+
+const assert = require('bsert');
+const AsyncEmitter = require('bevent');
+const rules = require('hsd/lib/covenants/rules');
+const {Network} = require('hsd');
+const {NodeClient, WalletClient} = require('hs-client');
+const NameState = require('hsd/lib/covenants/namestate');
+const random = require('bcrypto/lib/random');
+const {types} = rules;
+const {states} = NameState;
+
```

Comment:
Be consistent across the codebase regarding `NodeClient`. It is `node` in `lib/auctionrunner.js` and it is `nclient` in `lib/nodefactory.js`